sceptre/scipool/config/develop/sc-product-ec2-linux-docker.yaml

```diff
@@ -59,5 +59,5 @@ sceptre_user_data:
       Name: 'v1.4.0'
     - Description: 'Update Amazon Linux AMI {{ range(1, 10000) | random }}'
       Info:
-        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/master/ec2/sc-ec2-linux-docker.yaml'
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.4.3/ec2/sc-ec2-linux-docker.yaml'
```

Comment:
need to remove {{ range(1, 10000) | random }} from previous version as well.

Reply:
It was already removed in the dev account